helix-ui/src/main/java/org/apache/helix/ui/api/IdealStateSpec.java

```diff
@@ -0,0 +1,94 @@
+package org.apache.helix.ui.api;
+
+import org.apache.helix.model.IdealState;
+
+import java.util.List;
+
+public class IdealStateSpec {
```

Comment:
Yeah... I have mixed thoughts on this kind of stuff (if you're writing a schema to be used only in a Java project, why not just write the pojo). But IntelliJ generates everything faster than it takes to really think about it, so usually just do that.